File: webservice-conclave/webservice-conclave/client/src/main/java/com/r3/conclave/sample/client/FileLoader.java
package com.r3.conclave.sample.client;

import java.io.File;
import java.util.Scanner;

public class FileLoader {

    public static String OBJECT_SEPARATER = "\n";
    public static String RECORD_SEPARATER = ";";
    public static String ELEMENT_SEPARATER = ",";

    public static String loadDataFromFile(String file) {
        try {
            StringBuilder stringBuilder = new StringBuilder();
            Scanner scanner = new Scanner(new File(file));

            while (scanner.hasNextLine()) {
                String line = scanner.nextLine();
                stringBuilder.append(line).append(RECORD_SEPARATER);
            }
            return stringBuilder.toString();
        } catch (Exception e){
            e.printStackTrace();
            return "";
        }
    }
}

File: corda-conclave/diabetes-cordapp/workflows/src/main/kotlin/com/template/flows/FileLoader.kt
package com.template.flows;

import java.io.BufferedReader
import java.io.File
import java.io.InputStreamReader
import java.util.*

object FileLoader {
    val OBJECT_SEPARATER = "\n"
    val RECORD_SEPARATER = ";"
    val ELEMENT_SEPARATER = ","

    fun loadDataFromResource(file: String): String {
        val inputStream = FileLoader::class.java.getResourceAsStream(file)
        val streamReader = InputStreamReader(inputStream)
        val reader = BufferedReader(streamReader)

        val stringBuilder = StringBuilder()
        var line = ""
        while (reader.readLine().also { line = it } != null) {
            stringBuilder.append(line).append(RECORD_SEPARATER)
        }
        return stringBuilder.toString()
    }

    fun loadDataFromFile(file: String): String {
        val stringBuilder = StringBuilder()
        Scanner(File(file)).use { scanner ->
            while (scanner.hasNextLine()) {
                val line = scanner.nextLine()
                stringBuilder.append(line).append(RECORD_SEPARATER)
            }
        }
        return stringBuilder.toString()
    }
}

